src/book/03-guides/01-main-codebase/02-maintaining-subsystems.mdx

```diff
@@ -10,6 +10,17 @@ authors:
 
 This page details how to maintain various subsystems within the main codebase.
 
+## Odometry
+
+### Webots Live Error Tracking
+
+The `nugus_controller` in Webots has functionality to send live odometry ground truth data to the robot. To use this
+
+1. [Set up Webots and compile the NUWebots code](/guides/tools/webots-setup) (you do not need the RoboCup set up).
+2. Run the `kid.wbt` world in Webots.
+3. Run `webots_keyboardwalk` in NUbots. [See the Getting Started page to set up the NUbots codebase](/guides/main/getting-started).
+4. Run NUsight (or PlotJuggler) and observe the graphs for the torso to world transform's rotational and translational error.
```

Comment:
We can probably link to the PlotJuggler README here for how to use it, once that PR is merged.

Reply:
Sounds good. Wasn't sure whether to mention PlotJuggler, but it will probably get merged before this.

```suggestion
4. Run [NUsight](/guides/main/getting-started#get-build-and-run-nusight) or [PlotJuggler](https://github.com/NUbots/NUbots/tree/main/module/network/PlotJuggler/README.md) and observe the graphs for the robot's prediction, the ground truth and the error of the torso's real rotation and translation relative to world.
```